src/client/home/components/StatisticsSliver.tsx

```diff
@@ -79,17 +70,19 @@ const useStyles = makeStyles((theme) =>
   }),
 )
 
-const StatisticsSliver = (props) => {
+const StatisticsSliver: FunctionComponent = () => {
   const classes = useStyles()
-  const { statistics } = props
+  const dispatch  = useDispatch()
+  const loadStats = () => dispatch(homeActions.loadStats())
+  const statistics = useSelector((state: GoGovReduxState) => state.home.statistics)
   const { userCount, linkCount, clickCount } = statistics
+  // ensure number will never be null
   const statisticsToShow = [
-    { label: 'PUBLIC OFFICERS ONBOARD', number: userCount },
-    { label: 'SHORT LINKS CREATED', number: linkCount },
-    { label: 'CLICKS', number: clickCount },
+    { label: 'PUBLIC OFFICERS ONBOARD', number: userCount || 0 },
+    { label: 'SHORT LINKS CREATED', number: linkCount || 0 },
+    { label: 'CLICKS', number: clickCount || 0 },
```

Comment:
this sounds like a problem that should have been solved in the redux level, like both in the initial state of the reducer and in the api-call handlers that ensure to never set it to null

does it make more sense to solve this nullability problem in the redux initial state and api-call-result-handler rather than here?